frontend/apps/reader/modules/readerview.lua

```diff
@@ -782,6 +773,8 @@ function ReaderView:onReadSettings(config)
     self.page_scroll = page_scroll == 1 and true or false
     self.highlight.saved = config:readSetting("highlight") or {}
     self.page_overlap_style = config:readSetting("page_overlap_style") or G_reader_settings:readSetting("page_overlap_style") or "dim"
+    self.page_gap.height = Screen:scaleBySize(config:readSetting("kopt_page_gap_height") or
+    G_reader_settings:readSetting("kopt_page_gap_height") or 8)
```

Comment:
(nitpick: I'd like a bit more indentation on that 2nd line, to avoid confusion and make it clear it's continued from previous)